recipes/spades/meta.yaml

```diff
@@ -38,22 +30,22 @@ test:
     - metaspades.py --version
     - plasmidspades.py --version
     - spades.py --test && rm -rf spades_test
+    - spades-kmercount -h
 
 about:
   home: http://cab.spbu.ru/software/spades/
   license: GPLv2
   license_family: GPL
-  license_file: LICENSE  # [linux]
-  license_file: share/spades/LICENSE  # [osx]
+  license_file: share/spades/LICENSE
   summary: |
     SPAdes (St. Petersburg genome assembler) is intended for both standard isolates and single-cell MDA bacteria assemblies.
   dev_url: https://github.com/ablab/spades
-  doc_url: http://cab.spbu.ru/files/release3.11.0/manual.html
+  doc_url: http://cab.spbu.ru/files/release{{version}}/manual.html
 extra:
-  doi: 10.1089/cmb.2012.0021
   recipe-maintainers:
     - druvus
     - notestaff
   identifiers:
     - biotools:Spades
     - doi:10.1089/cmb.2012.0021
+    - doi:10.1101/gr.213959.116
```

Comment:
This is metaspades isn't it?

Reply:
@bgruening Yes, which is part of the spades package.